### PR TITLE
Feature/integrate-sdk

### PR DESCRIPTION
**Pull request by Ahmedhossamdev**
This PR contains the integration of the SDK in the arrivalsAndDepartures and stop endpoints

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
 	},
 	"type": "module",
 	"dependencies": {
-		"@fortawesome/free-regular-svg-icons": "^6.6.0"
+		"@fortawesome/free-regular-svg-icons": "^6.6.0",
+		"onebusaway-sdk": "^0.1.0-alpha.31"
```
Comment (aaronbrethorst) [inline package.json:40]: 🔥

File: src/lib/RestAPI/arrivalsAndDeparturesForStop.js [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
+});
+
 export default async function (stopID) {
-	const apiURL = `${baseURL}/api/where/arrivals-and-departures-for-stop/${stopID}.json?key=${apiKey}`;
-	const response = await fetch(apiURL);
+	const response = await oba.arrivalAndDeparture.list(stopID);
 
-	if (!response.ok) {
+	if (response.code !== 200) {
 		error(500, 'Unable to fetch arrivals-and-departures-for-stop.');
```
Comment (aaronbrethorst) [inline src/lib/RestAPI/arrivalsAndDeparturesForStop.js:16]: does the `error()` method here function function like an exception? i.e. does it interrupt control flow?

**Review by aaronbrethorst — CHANGES_REQUESTED** [review]
Looks great! Couple questions for you before merging!

File: src/lib/RestAPI/arrivalsAndDeparturesForStop.js (hunk 2 above)
Comment (Ahmedhossamdev) [reply src/lib/RestAPI/arrivalsAndDeparturesForStop.js:16]: image

I don't know, I try it now, it seems not interrupt the flow, but when I enter stops route endpoint the error message appears in the top

image

https://learn.svelte.dev/tutorial/error-basics

but according to svelte docs the `expected error message is shown to the user`.

**Review by aaronbrethorst — APPROVED** [review]
super cool. I'm excited to see this get used across the codebase. I'm going to merge it now and you can continue adding new support for other endpoints

**Comment by aaronbrethorst** [comment]
FYI @tarunsinghofficial - make sure you pull main and do an nPm install :)

**Pull request merged**